recipe/recipe.yaml

```diff
@@ -0,0 +1,88 @@
+context:
+  name: pyright
+  version: 1.1.388
+
+recipe:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/pyright-${{ version }}.tar.gz
+  sha256: 0166d19b716b77fd2d9055de29f71d844874dbc6b9d3472ccd22df91db3dfa34
+
+build:
+  number: 1
+
+
+outputs:
+  - package:
+      name: ${{ name|lower }}-node
+    build:
+      skip: [ppc64le]
+      script:
+        file: build_node${{ '.bat' if win else '.sh' }}
+    requirements:
+      build:
+        - pnpm
+        - pnpm-licenses
+    about:
+      license: MIT
+      license_file: third-party-licenses.txt
+
+  - package:
+      name: ${{ name|lower }}
+    build:
+      skip: [ppc64le]
+      script:
+        file: build_python${{ '.bat' if win else '.sh' }}
+      python:
+        entry_points:
+          - pyright=pyright.cli:entrypoint
+          - pyright-python=pyright.cli:entrypoint
+          - pyright-langserver=pyright.langserver:entrypoint
+          - pyright-python-langserver=pyright.langserver:entrypoint
+
+    requirements:
+      build:
+      - if: build_platform != target_platform
+        then: python
+      - if: build_platform != target_platform
+        then: cross-python_${{ target_platform }}
+      host:
+      - python
+      - setuptools
+      - pip
+      run:
+      - python
+      - nodejs
+      - nodeenv >=1.6.0
+      - typing-extensions >=4.1
+      - ${{ pin_subpackage(name ~ '-node', upper_bound='x.x.x') }}
```

Comment:
if we actually build the package multiple times i think we can keep `exact=true`